### ISSUE TITLE
cant generate output png

### ISSUE THREAD

**Issue by MyraBaba** [satoshi-ikehata/Universal-PS-CVPR2022#5]
It could be a dumb question :) but I cant generate 3D ish image from 👍 

python source/main.py --session_name session_test --mode Test --test_dir owml/  --pretrained checkpoint/

only output:

> 
> `Namespace(session_name='session_test', training_dir='DefaultTraing', test_dir='owml/', mode='Test', agg_type='Transformer', batchsize=4, outdir='output', pretrained='checkpoint/', num_agg_enc=3, min_nimg=2, num_samples=2500, lr=0.0001, args=None, lr_scheduler='step', lr_init_scale=1.0, encoder_imgsize=256)
> cuda
> 2.2.0+cu121
> Encoder Backbone  is SwinTransformer
> optimizer (AdamW) lr=0.0001
> step late scheduler x0.8 decay
> Transformer (#Enc 3, Dimhidden 384, DimFeedforward 1024, Norm True, Dropout 0.1)
> optimizer (AdamW) lr=0.0001
> step late scheduler x0.8 decay
> optimizer (AdamW) lr=0.0001
> step late scheduler x0.8 decay
> Session session_test: #images >= 2, #samples = 2500
> Load checkpoint/attenuper_enc.pytmodel
> Load checkpoint/attenuper_agg.pytmodel
> Load checkpoint/attenuper_pred.pytmodel
> Load checkpoint/attenuper_enc.scheduler
> Load checkpoint/attenuper_agg.scheduler
> Load checkpoint/attenuper_pred.scheduler
> Load checkpoint/attenuper_enc.optimizer
> Load checkpoint/attenuper_agg.optimizer
> Load checkpoint/attenuper_pred.optimizer
> Pretrained Model Loaded! initial LR is 2.6214400000000015e-05
> Initialize DefaultTraing
> Number of Train set is 0
> Initialize owml/
> Number of Test set is 0
> learning rate updated  0.00003 -> 0.00003
> GLC Encoder
> 
> 
> # parameters: 63701034
> Decoder:Aggregation
> # parameters: 5625350
> Decoder:Prediction
> # parameters: 74499
> Run 1-th epoch
> session_test, TestMode`

**Comment by satoshi-ikehata** [comment]
"Number of Test set is 0 " indicates that the test data couldn't be found. Please check your file path.
I assume that "owml/" should be "owl/".
